### DISCUSSION TITLE
Data Kiosk Economics report - Time period available

### DISCUSSION THREAD

**Discussion by kevnhowe** [amzn/selling-partner-api-models#4242] [Q&A] [answered]
What is the maximum number of days in the past I can query using the Data Kiosk Economics report? Anything I've run around a year and older returns no records. I get dates and asins, but no transaction data. I don't see any guidance on this in the documentation.

**Comment by chrisskhoury** [comment, marked as answer]
Hello @kevnhowe :)

Currently, the Data Kiosk Economics Dataset has specific limitations on the range of historical data you can query. Here are the details:

**Sales Data:** You can query sales data for up to 6 months in the past.
**Fees Data:** You can query fees data for up to 2 years in the past.

This extended range is provided to help users analyze long-term trends and performance metrics related to fees and our team is working on extending the availability of sales data to cover a 2-year period as well. 

Also, thanks for the feedback and pointing out that this information is not currently mentioned in the documentation. We understand the importance of comprehensive dataset documentation and will work on updating the documentation to include these details and more specifics to each dataset.

Cheers,
Chris
